backend/src/common/enums/availability-type.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum AvailabilityType {
+  AVAILABILITY,
```

Comment:
AVAILABLE?

I think we 'UNAVAILABLE' as well? (ie: not deployed but they're on holiday or something)

Reply:
Well I now know that AVAILABLE is default, so it should be UNAVAILABLE and DEPLOYED I think

Reply:
Should we set the default value automatically for a resource?

Reply:
(we could also include this one later once we have fully spec'd out the scheduler.) I am not including availability on the dash yet since I see it as a dependency of the scheduler...

Reply:
I'm not in full agreement that it should be a dependency of the scheduler, but it could certainly be after those discussions.

The way I have it now, we wouldn't have an entry for the user for the table if they are AVAILABLE since that is the default. On the frontend, we can just render it that way (days with missing values are AVAILABLE). To avoid from having an entry on this table for every single day of the month per user.